builder/amazon/common/ssh.go

```diff
@@ -28,6 +28,10 @@ func SSHHost(e ec2Describer, sshInterface string, host string) func(multistep.St
 			return host, nil
 		}
 
+		if sshInterface == "session_manager" {
+			return "127.0.0.1", nil
```

Comment:
Should this be "localhost"?

Reply:
If this works, I think it's probably fine? In an ideal situation they're equivalent, right?

Reply:
I'm thinking in terms of IPv6 the address might be `::1`, but not entirely sure. Maybe we can default to this for now  and add an option to override if needed.

Reply:
So on my initial review I wondered about this and the answer internet gave me is `127.0.0.1` is better in this situation.